### DISCUSSION TITLE
Bei abschalten des DPL geht HM600 in Standby - forever

### DISCUSSION THREAD

**Discussion by 3joern** [hoylabs/OpenDTU-OnBattery#877] [Q&A] [answered]
Ahoi Zusammen,
ich habe da ein Phänomen, das ich nicht zuordnen kann, ob das an meinen Einstellungen liegt oder ob da was anderes schief läuft. Wenn ich den DPL ausschalte, geht der WR in Standby und wacht nicht mehr auf, bis ich den DPL wieder aktiviere, hier mal infos aus der Konsole (Zeile 71):

https://pastebin.com/EugzjbV6

Hier die Einstellungen. Das untere Leistungslimit hat hier keine Auswirkung, auch bei 50W das gleiche Verhalten. 
![Bildschirmfoto 2024-04-15 um 17 55 26](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/47fb6c66-7545-4d1a-8b6b-4a82c6299b23)

Akku läuft auf dem 2. String einfach als Solarmodul, weil der sich selbst unter 10% ausklinkt oder bei über 100% auf Passthrough geht.
![Bildschirmfoto 2024-04-15 um 17 57 51](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/cf96a2b9-b620-4986-a0a8-6c43cfe48cb2)

**Comment by spcqike** [comment, marked as answer]
Wenn man den DPL im Web UI deaktiviert, wird auch der wechselrichter in den standby geschickt. Das ist (glaube ich) schon immer so und gewollt. 

Du kannst / musst den wechselrichter danach Manuell starten. Einfach im Web UI im liveview am wechselrichter.

**Reply by schlimmchen** [reply]
Korrekt. Falls er sich danach nicht per Web UI oder per MQTT starten ließe, wäre das ein Fehler. Aber wenn man den DPL abstellt, wird der zu regelnde WR in den Standby versetzt, bevor der DPL dann nichts mehr tut.

Danke @3joern für die vorbildliche Beschreibung mit Screenshots und Logs 💪

**Comment by 3joern** [comment]
Habt Dank :)
